Sources/SwiftProtobuf/FieldNameMap.swift

```diff
@@ -94,38 +62,47 @@ public struct FieldNameMap: ExpressibleByDictionaryLiteral {
   public enum Names {
 
     /// The proto name and the JSON name are the same string
-    case same(proto: String)
+    case same(proto: StaticString)
 
     /// The JSON and text names are different and not derivable from each other.
-    case unique(proto: String, json: String)
+    case unique(proto: StaticString, json: StaticString)
 
     // TODO: Add a case for JSON names that are computable from the proto name
     // using the same algorithm implemented by protoc; for example,
     // "foo_bar" -> "fooBar". This allows us to only store the string pointer
     // in the payload once.
 
     /// Returns the proto (and text format) name in the bundle.
-    public var protoName: String {
+    public var protoStaticStringName: StaticString {
       switch self {
       case .same(proto: let name): return name
       case .unique(proto: let name, json: _): return name
       }
     }
 
+    /// Returns the proto (and text format) name in the bundle.
+    public var protoName: String {
```

Comment:
None of our code uses this, so we could drop them.

If we did open up `FieldNameMap` for users, then I would prefer to have `String`-based access rather than exposing the `UBP` ones.